### PR TITLE
Create StartANewReactApp.md

### LINKED ISSUE #2914: Process for starting a react application not ready for production?
Thinking about the Service and Facility Locator James is working on, as well as Secure Messaging, which we'll be starting on next week...

@crwallace it looks like Prescriptions was specifically added to [build.js](https://github.com/department-of-veterans-affairs/vets-website/blob/master/script/build.js#L96) to say that it's not ready for production. When starting other applications, is adding to this file a step we should take in our first PR?  Are there other things we should do when starting applications to make sure they aren't accidentally deployed?

### PR DESCRIPTION
**Pull request by crwallace**
closes #2914.

@akainic -- do you mind reviewing this and letting me know if there is anything I missed?

### CODE REVIEW

File: docs/StartANewReactApp.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# How to Start a New ReactJS Project
+
+
+1. Create a directory for your project inside of `vets-website/src/js/`. 
+2. Create an entry file in your project directory using the project name to prefix the filename. Example: `vets-website/src/js/[project-name]/[project-name]-entry.jsx`.
+3. Add the entry to `vets-website/config/webpack.config.js`.
+4. Create a directory and a `.md` file inside of `vets-website/content/pages/` for your project.
```
Comment (akainic) [inline docs/StartANewReactApp.md]: Do you want to specify `index.md`?

File: docs/StartANewReactApp.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# How to Start a New ReactJS Project
+
+
+1. Create a directory for your project inside of `vets-website/src/js/`. 
+2. Create an entry file in your project directory using the project name to prefix the filename. Example: `vets-website/src/js/[project-name]/[project-name]-entry.jsx`.
```
Comment (akainic) [inline docs/StartANewReactApp.md]: If you're using education benefits for all the examples here, I might move the line you have above with `[project-name]` in it before the example and have the example actually be the `edu-benefits` entry file name. So:
```
Create an entry file for your project directory using the project name to prefix the filename: `vets-website/src/js/[project-name]/[project-name]-entry.jsx`. Example: `vets-website/src/js/edu-benefits/edu-benefits-entry.js`.
```

**Comment by akainic** [comment]
Woo hoo thanks for putting this together! I don't think anything else is missing.

**Comment by crwallace** [comment]
@akainic -- Thanks so much for taking a look at this!

**Pull request merged**